doc/doc/download.rst.j2

```diff
@@ -453,24 +453,24 @@ Github.
 
    `Nuitka 0.6.18.4 Python3.9 64 bit MSI <https://nuitka.net/releases/Nuitka-6.1.184.win-amd64.py39.msi>`__
 
-.. |DEBIAN_LOGO| image:: images/debian.png
+.. |DEBIAN_LOGO| image:: ../../images/debian.png
```

Comment:
I think I saw you don't have the folder structure in the `intl` and therefore I would be surprised if that then works for that too. I think, even if the folder structure right now is bad, probably should need to have `doc` and `pages` there too now?

Reply:
Make sure `intl/start/download.rst` has the same file depth as `doc/doc/download.rst`.

Reply:
Ah, but please do not invent your own directory names, when I clean up the structure, it should be same, so if possible rename `start` to `doc` then?

Reply:
I agree.

Reply:
Resolving this, but note, before merging I would like to do that. I was not seeing `pages` directory, only `doc`. So it's not an exact mirror. I think during my switcher implementation, pages will only switch when there is a 1:1 filename mapping for a language.